.travis.yml

```diff
@@ -0,0 +1,6 @@
+language: node_js
+node_js:
+  - "6"
```

Comment:
I see Travis picked 6 up from `.travis.yml` and did `nvm install 6`. Maybe we could make it do `nvm install node` (like in the README, gets the latest version) instead? 

BTW, for the time being atleast, v6 and v7 sound good, although that would double the time it takes, so we should really think about it. If we decide to go with two versions, we can switch to v7 and v8 later.